app/services/wds-fetch.js

```diff
@@ -1,11 +1,14 @@
 import Service from '@ember/service';
 import fetch from "fetch";
 
+// FIXME get rid of wdsFetch service and use ember-fandom fetch service
 export default Service.extend({
 
 	servicesDomain: 'https://services.wikia.com/',
 
-	fetch(path, options) {
+	fetch(path, requestOptions) {
+		const options = Object.assign({ credentials: 'include' }, requestOptions);
```

Comment:
to make it work in IE11
you can use https://www.emberjs.com/api/ember/release/functions/@ember%2Fpolyfills/assign